CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# dbt_zendesk v0.22.0
+
+This release includes the following updates from pre-releases `v0.21.0-a1`, `v0.21.0-a2`, `v0.22.0-a1`, and `v0.22.0-a2`:
+
+## Schema Updates
+
+**4 total changes • 0 possible breaking changes**
+| **Data Model** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| [stg_zendesk__ticket_chat](https://fivetran.github.io/dbt_zendesk_source/#!/model/model.zendesk_source.stg_zendesk__ticket_chat) | New Model |   |   |  Uses `ticket_chat` source table  |
+| [stg_zendesk__ticket_chat_tmp](https://fivetran.github.io/dbt_zendesk_source/#!/model/model.zendesk_source.stg_zendesk__ticket_chat_tmp) | New Temp Model |   |   |  Uses `ticket_chat` source table  |
+| [stg_zendesk__ticket_chat_event](https://fivetran.github.io/dbt_zendesk_source/#!/model/model.zendesk_source.stg_zendesk__ticket_chat_event) | New Model |   |   | Uses `ticket_chat_event` source table   |
+| [stg_zendesk__ticket_chat_event_tmp](https://fivetran.github.io/dbt_zendesk_source/#!/model/model.zendesk_source.stg_zendesk__ticket_chat_event_tmp) | New Temp Model |   |   | Uses `ticket_chat_event` source table   |
+
+## Feature Updates:
+- **Messaging Chat Channel Support:** Incorporated the `ticket_chat` and `ticket_chat_event` source tables to properly capture tickets created via `chat` and `native_messaging` channels in SLA Policy and other ticket metrics. ([PR #187](https://github.com/fivetran/dbt_zendesk/pull/187))
+- (Upstream in [zendesk_source](https://github.com/fivetran/dbt_zendesk_source/blob/main/CHANGELOG.md)) Handled inconsistent formatting of `ticket_chat_event.actor_id` to safely cast it as a bigint. ([Source PR #63](https://github.com/fivetran/dbt_zendesk_source/pull/63))
+- Added the `using_ticket_chat` variable to enable/disable `ticket_chat` and `ticket_chat_event` staging models and transformations. ([PR #187](https://github.com/fivetran/dbt_zendesk/pull/187))
+  - For Fivetran Quickstart users, `using_ticket_chat` is dynamically set based on the presence of the `ticket_chat` and `ticket_chat_event` source tables.
+  - For other users, `using_ticket_chat` is set to **False** by default. To change this and enable the ticket chat models, add the following configuration (see [README](https://github.com/fivetran/dbt_zendesk?tab=readme-ov-file#step-4-enabledisable-models-for-non-existent-sources) for details).
+
+```yml
+vars:
+  zendesk_source:
+    using_ticket_chat: True
+  zendesk:
+    using_ticket_chat: True
+```
```

Comment:
This to me is reading that both need to be set--is that the case?

Reply:
It can also be set globally, but I did it this way since the variable names are generic and don't have a `zendesk__` prefix

Reply:
Oh I get it. I would still do just one `using_ticket_chat: True` since this is a Zendesk-only update and won't roll up anywhere right? I can imagine users thinking both are needed in that format, but I'll let you make the final call!

Reply:
True -- I'll switch it 👍